Patch-Source: https://github.com/aperezdc/ngx-fancyindex/commit/674e99547d21b8bd9a36dc5d8689d08632d3a34d
--
From 674e99547d21b8bd9a36dc5d8689d08632d3a34d Mon Sep 17 00:00:00 2001
From: Adrian Perez de Castro <aperez@igalia.com>
Date: Thu, 19 Jan 2023 20:51:08 +0200
Subject: [PATCH] Add missing entry in long weekday names array

The entry for Saturday was missing, as pointed out in the bug report.
While at it, add a test case.

Fixes #157

diff --git a/ngx_http_fancyindex_module.c b/ngx_http_fancyindex_module.c
index bfccdc8..ed349d5 100644
--- a/ngx_http_fancyindex_module.c
+++ b/ngx_http_fancyindex_module.c
@@ -37,7 +37,7 @@ static const char *short_weekday[] = {
     "Mon", "Tue", "Wed", "Thu", "Fri", "Sat", "Sun",
 };
 static const char *long_weekday[] = {
-    "Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Sunday",
+    "Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday", "Sunday",
 };
 static const char *short_month[] = {
     "Jan", "Feb", "Mar", "Apr", "May", "Jun",
